### PR TITLE
adds the except exercise files

### PR DESCRIPTION
**Pull request by jkrecek3**

### CODE REVIEW

File: students/jim_krecek/lesson05/mailroom_p3.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import sys
+
+data = {"Nate Secinaro":[12,44,9],"Jess Reid":[3],"Pat Carrier":[65,41],"Zac Fisher":[13,4,4],"Jim Krecek":[3,33,57]}
+
+prompt = "\nWhat do you want to do?\n -1 Send a Thank You\n -2 Create a Report\n -3 Send letters to all donors\n -4 Quit\n>>> "
+
+def send_ty():
+    askname = input("\nEnter the full name of the donor or type 'list' to see the list of donors:\n>>> ")
```
Comment (charlesrobison) [inline students/jim_krecek/lesson05/mailroom_p3.py]: You'll want all variable names to be snake_case, just like your function names

File: students/jim_krecek/lesson05/mailroom_p3.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            data[askname].append(int(askmoney))
+        emailtext = "\nDear {},\n\nThank you for your generous donation of ${}.\n\nSincerely, Jim".format(askname,askmoney)
+        print(emailtext)
+            
+
+def create_report():
+    firstline = "{:<25}| {:<12}| {:<10}| {:>12}".format("Donor Name", "Total Given", "Num Gifts", "Average Gift")
+    print(firstline)
+    print("-"*len(firstline))
+    sorted_data = sorted(((sum(value),len(value),key) for (key,value) in data.items()), reverse=True)
```
Comment (charlesrobison) [inline students/jim_krecek/lesson05/mailroom_p3.py]: Nice comprehension

File: students/jim_krecek/lesson05/mailroom_p3.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+ 
+def main():
+    main_menu = {'1': send_ty,'2': create_report,'3':send_all,'4':quit}
+    while True:
+        response = input(prompt)
+        menu_function = main_menu.get(response)
+        try:
+            menu_function()
+        except TypeError:
+            print("\n'{}' is not a valid entry. Please try again.".format(response))
```
Comment (charlesrobison) [inline students/jim_krecek/lesson05/mailroom_p3.py]: Good use of exception - are there other opportunities for other exceptions in this code?

Comment (jkrecek3) [reply students/jim_krecek/lesson05/mailroom_p3.py]: thanks! after looking through a bit more, I don't think there are other exception opportunities.

**Pull request merged**